### ISSUE TITLE
Accessibility Issues with FileExplorer for Screen Readers

### ISSUE THREAD

**Issue by MesterPerfect** [omeryanar/FileExplorer#78]
There are major accessibility issues with FileExplorer when using screen readers, such as NVDA. For example, when browsing the application, I cannot read the name of any item. Additionally, I cannot access the settings or navigate between different tabs.

**Comment by omeryanar** [comment]
FileExplorer doesn't support screen readers, sorry.
